src/Illuminate/Http/Middleware/TrustHosts.php

```diff
@@ -59,6 +59,18 @@ protected function shouldSpecifyTrustedHosts()
                ! $this->app->runningUnitTests();
     }
 
+    /**
+     * Get a regular expression matching the application URL.
+     *
+     * @return string|null
+     */
+    protected function onlyApplicationUrl()
+    {
+        if ($host = parse_url($this->app['config']->get('app.url'), PHP_URL_HOST)) {
```

Comment:
this would break our application, we do use app url for a default url but our application is hosted on multiple domains.

Reply:
This is merely an additive method that will do nothing unless you use it in your application.